tests/unit/tf/blocks/core/test_tabular.py

```diff
@@ -1,28 +1,42 @@
-import merlin.models.tf as ml
```

Comment:
i understand that `mm` is now the preferred import alias, and when we're changing imports as we go.

Reply:
Yes, we go with `mm`.